FewCore/Element.swift

```diff
@@ -116,7 +118,9 @@ public class Element {
 				compareAndSetAlpha(view, alpha)
 			}
 			
-			realizedSelf.element = self
+			if old.frame.size != frame.size {
```

Comment:
At this point `frame.size` won't have been set, so this'll almost always get hit.